packages/netlify-cms-widget-markdown/src/serializers/remarkImagesToText.js

```diff
@@ -15,8 +15,8 @@ export default function remarkImagesToText() {
         child.children.length === 1 &&
         child.children[0].type === 'image'
       ) {
-        const { alt = '', url = '', title = '' } = child.children[0];
-        const value = `![${alt}](${url}${title ? ' title' : ''})`;
+        const { alt, url, title } = child.children[0];
+        const value = `![${alt || ''}](${url || ''}${title ? ' title' : ''})`;
```

Comment:
i've kept the implementation of `title` prop unchanged.
but it probably should look like this: ``${title ? ' ' + title : ''}``

Reply:
Ha, yeah that's busted. Can you just drop the title stuff for now? It doesn't really seem to be supported elsewhere in our serialization process, and no one has complained that it's completely broken.

Reply:
sure! will do (:

Reply:
done @erquhart!